src/emulator/functionsEmulator.ts

```diff
@@ -623,6 +623,13 @@ export class FunctionsEmulator implements EmulatorInstance {
               definition.schedule
             );
             break;
+          case Constants.SERVICE_EVENTARC:
```

Comment:
It seems like this won't ever be Constants.SERVICE_EVENTARC becuase getFunctionService(definition) doesn't return that value

Reply:
Good catch, thanks!